sdk/python/feast/feature_view.py

```diff
@@ -121,15 +121,28 @@ def __init__(
         """
         _features = features or []
 
+        if stream_source is not None and isinstance(stream_source, PushSource):
+            assert stream_source.batch_source is not None
+            assert isinstance(stream_source.batch_source, DataSource)
+            self.batch_source = stream_source.batch_source
+        else:
+            assert batch_source is not None
+            self.batch_source = batch_source
+
+        if not self.batch_source:
```

Comment:
this error should be switched with the above assert I think

Reply:
done